### PR TITLE
Bump some stuff

### PR DESCRIPTION
**Pull request by tbreloff**

### CODE REVIEW

File: Plots/versions/0.9.4/requires [hunk 1]
```diff
@@ -0,0 +1,8 @@
+julia 0.5-
+RecipesBase
+PlotUtils
+Reexport
+Compat
```
Comment (tkelman) [inline Plots/versions/0.9.4/requires]: should declare a minimum version dependency that implements the features you need

**Comment by tbreloff** [comment]
Hmm. I don't know if I'm even using Compat anymore.  Is this merge-blocking?

File: Plots/versions/0.9.4/requires (hunk 1 above)
Comment (tkelman) [inline Plots/versions/0.9.4/requires]: if you delete it here it should also be deleted in the package require

**Comment by tbreloff** [comment]
Any chance you can merge PlotThemes? I deleted compat but I also have
commits that add that as a dependency, so it would be great to include both
changes.

**Comment by tkelman** [comment]
new package registrations stay open 24 hrs for initial feedback. you don't need to change the sha here to remove the dependency as long as it didn't require any code changes

File: Plots/versions/0.9.4/requires (hunk 1 above)
Comment (tkelman) [inline Plots/versions/0.9.4/requires]: Should be deleted from https://github.com/tbreloff/Plots.jl/blob/master/REQUIRE before the next tag.

**Comment by tbreloff** [comment]
Thanks Tony. I can tag again after PlotThemes is merged.

**Pull request merged**